### PR TITLE
Add Nick and remove Emmanuel from site-admin-emails

### PR DESCRIPTION
**Pull request by nickscialli-usds**
## Related Issue or Background Info

- Adding myself as an admin and removing Emmanuel for all environments

### CODE REVIEW

**Review by timothybest-usds — APPROVED** [review]

File: backend/src/main/resources/application-cloud.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ simple-report:
     - neil.s.sharma@omb.eop.gov
     - wsullivan@cdc.gov
     - jeremy.a.zitomer@omb.eop.gov
-    - eapau@cdc.gov
+    - nicholas.scialli@usds.dhs.gov
```
Comment (benwarfield-usds) [inline backend/src/main/resources/application-cloud.yaml:21]: Oh, we should probably delete this file￼. (No longer used.)

**Review by benwarfield-usds — APPROVED** [review]
LGTM

File: backend/src/main/resources/application-cloud.yaml (hunk 1 above)
Comment (timothybest-usds) [reply backend/src/main/resources/application-cloud.yaml:21]: removed in https://github.com/CDCgov/prime-simplereport/pull/789. The PR could use a review

Comment (benwarfield-usds) [reply backend/src/main/resources/application-cloud.yaml:21]: Oh, I somehow thought that had already been merged. Will jump on over there, yes. 🙂

**Pull request merged**